File: app/src/main/java/org/take365/Network/Models/StoryListItemModel.java
package org.take365.Network.Models;

import java.io.Serializable;

/**
 * Created by evgeniy on 08.02.16.
 */
public class StoryListItemModel implements Serializable {
    public AuthorModel[] authors;
    public int id;
    public StoryProgressModel progress;
    public int status;
    public String title;
    public String utl;
}

File: app/src/main/java/org/take365/Network/Models/StoryListItemModel.kt
package org.take365.Network.Models

import java.io.Serializable

/**
 * Created by evgeniy on 08.02.16.
 */
data class StoryListItemModel(
        var authors: List<AuthorModel>,
        var id: Int,
        var progress: StoryProgressModel,
        var status: Int,
        var title: String? = null,
        var url: String? = null
) : Serializable
